pisp_app/lib/Screens/account_dashboard.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:flutter/material.dart';
+import 'package:pispapp/MockData/account.dart';
+import 'package:pispapp/Screens/account_details.dart';
+
+class AccountDashboard extends StatefulWidget {
+  @override
+  _AccountDashboardState createState() => _AccountDashboardState();
+}
+
+class _AccountDashboardState extends State<AccountDashboard> {
+  List<Account> accounts = <Account>[];
+  List<String> androidVersionNames = <String>[];
+
+  @override
+  void initState() {
+    super.initState();
+
+    accounts = getMyDummyAccounts();
+    accounts = accounts.where((Account element) => element.linked).toList();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        body: Column(
+      children: <Widget>[
```

Comment:
There is also an indentation issue here because you don't use a trailing comma (can check my other comment)